### PR TITLE
[MNOE-470] Currency Selection (Customer / Tenant)

### PR DESCRIPTION
**Pull request by MAhsenArif**

### CODE REVIEW

**Comment by manu-d** [comment]
Same as https://github.com/maestrano/mno-enterprise/pull/629, can you leave only the english translation?

File: src/app/views/company/settings/organization-settings.directive.coffee [hunk 1]
```diff
@@ -52,6 +53,16 @@ DashboardOrganizationSettingsCtrl = ($scope, $window, MnoeOrganizations, Utiliti
     f = $scope.forms
     $scope.isChanged() && f.settings.$valid
 
+  $scope.isCurrencyChangeShown = ->
+    if MnoeOrganizations.role.isSuperAdmin()
+      $scope.availableBillingCurrencies()
```
Comment (ouranos) [inline src/app/views/company/settings/organization-settings.directive.coffee]: What the advantage of this vs:
```coffescript
$scope.isCurrencyChangeShown = -> MnoeOrganizations.role.isSuperAdmin()
$scope.currencies = MnoeConfig.availableBillingCurrencies()
```
?

File: src/app/views/company/settings/organization-settings.html [hunk 2]
```diff
@@ -19,6 +19,15 @@
         <input type="text" class="form-control" ng-model="model.name" required/>
       </div>
     </div>
+
+    <div class="form-group" ng-if="isCurrencyChangeShown()">
+      <label class="col-sm-3 control-label">{{ 'mno_enterprise.templates.dashboard.organization.settings.billing_currency' | translate }}</label>
+      <div class="col-sm-6">
+        <select ng-model="model.billing_currency" class="form-control">
+          <option ng-repeat="currency in currencies" value="{{currency}}">{{currency}}</option>
```
Comment (ouranos) [inline src/app/views/company/settings/organization-settings.html]: You can use one-way binding here (`{{::currency}`) as the list is static

File: src/app/views/company/settings/organization-settings.directive.coffee (hunk 1 above)
Comment (MAhsenArif) [reply src/app/views/company/settings/organization-settings.directive.coffee]: I added it like this as a form of secondary validation check to never load currencies if the user is not Super Admin.

Comment (ouranos) [reply src/app/views/company/settings/organization-settings.directive.coffee]: Ok fair enough. Maybe rename it to `$scope.getAvailableBillingCurrencies()` to make it more explicit?

**Comment by MAhsenArif** [comment]
@ouranos Feedback incorporated.

**Pull request merged**